packages/@aws-cdk/aws-cloudfront/lib/web-distribution.ts

```diff
@@ -347,9 +348,18 @@ export interface Behavior {
    * The signers are the account IDs that are allowed to sign cookies/presigned URLs for this distribution.
    *
    * If you pass a non empty value, all requests for this behavior must be signed (no public access will be allowed)
+   * @deprecated - We recommend using TrustedKeyGroups instead of TrustedSigners.
```

Comment:
Minor change in case to match the props.
```suggestion
   * @deprecated - We recommend using trustedKeyGroups instead of trustedSigners.
```

Reply:
Fixed.